### ISSUE TITLE
crash with GTNH NEI? :0

### ISSUE THREAD

**Issue by brandyyn** [kotmatross28729/Shader-fixer#9]
[fml-client-latest.log](https://github.com/user-attachments/files/18057626/fml-client-latest.log)

**Comment by kotmatross28729** [comment]
Interesting, this apparently happened due to the GTNH NEI update, I'll take a look when I can

**Comment by kotmatross28729** [comment]
Ah, now I see, this is a conflict between the rple mixin and the shader fixer. Although, this should not happen (the priority of the shader fixer mixin is 999, and rple is 1000, since rple has a higher priority, it is applied earlier) and mixins should be skipped, but still.


I made "safe" versions of mixins that don't contain `Redirect`, which will be loaded if RPLE is installed.
Because of this, the `GL_LIGHTING` and `GL_BLEND` leak fix will not work, but the shader fix itself will work

although, I’ll probably do a PR in GTNH NEI rather than carry these Redirects


In short, try:
[libs.zip](https://github.com/user-attachments/files/18058725/libs.zip)
